uiowa/uiowa
# Admissions student profile views block update

Pulled out from #6054 

**Note:** When this is rolled out, it will cause a slight visual regression and will require updating  https://admissions.uiowa.edu/international-student-ambassadors, https://admissions.uiowa.ddev.site/california-hawkeyes and https://admissions.uiowa.ddev.site/texas-hawkeyes as soon as the deployment is done.

# How to test
```
ddev blt ds --site admissions.uiowa.edu && ddev drush @admissions.local uli node/7331/layout
```

- Remove existing "Student Profiles Image Grid" block
- Add "Student Profiles Card Grid" block, setting the list format to Grid, and the grid layout to 3 column
- Save the layout
- Compare with https://admissions.uiowa.edu/international-student-ambassadors
- Ensure no changes on https://admissions.uiowa.ddev.site/student-profiles vs https://admissions.uiowa.ddev.site/student-profiles
- Confirm that image grid on student profile pages are still working https://admissions.uiowa.ddev.site/student-profiles/ainsley-hiestand

## Page titles

- Test that these page titles have gold highlighted background.   The student profile should be using Zilla, and the other two are using Antonio.

https://admissions.uiowa.ddev.site/academics/2plus2/anthropology-northeast-iowa-community-college
https://admissions.uiowa.ddev.site/academics/accounting
https://admissions.uiowa.ddev.site/student-profiles/sophia-restiffe-favoretto
